app/services/duplicate_application.rb

```diff
@@ -31,6 +31,7 @@ def duplicate
       new_application_form.update(
         feature_restructured_work_history: true,
         work_history_completed: false,
+        work_history_status: new_application_form.application_work_experiences.present? ? 'can_complete' : nil,
```

Comment:
maybe this should be `can_not_complete` actually?

Reply:
not sure to be honest. Doesn't `can_not_complete` imply that the candidate answered the question and explicitly said 'no'?

Reply:
Yea good point. Will give it a quick test locally with an application that hasn't provided one using the previous flow just to double check behaviour

Reply:
So i think this should actually be set to can_not_complete here because with the old flow they have to provide an explanation if they don't have a work history.

Reply:
![image](https://user-images.githubusercontent.com/42515961/135473992-b6ad1d50-84d0-4a43-9664-7576d05fd603.png)

none have been carried over with no work history as of yet so no backfill is needed

Reply:
Just realised my query was wrong. There are 5